ui/round/src/view/replay.ts

```diff
@@ -115,7 +117,7 @@ function renderMoves(ctrl: RoundController): MaybeVNodes {
   const els: MaybeVNodes = [],
     curPly = ctrl.ply;
   for (let i = 0; i < pairs.length; i++) {
-    els.push(h(indexTag, i + 1 + ''));
+    els.push(h(indexTag, firstPly + indexOffset + ''));
```

Comment:
I think that will set the same turn to all moves, since `i` is no longer used.

Reply:
This is what happens when you make cosmetic changes _after_ verifying your fix.  ;p